nwbinspector/register_checks.py

```diff
@@ -78,7 +78,24 @@ class InspectorMessage:
 # TODO: neurodata_type could have annotation hdmf.utils.ExtenderMeta, which seems to apply to all currently checked
 # objects. We can wait and see how well that holds up before adding it in officially.
 def register_check(importance: Importance, neurodata_type):
-    """Wrap a check function to add it to the list of default checks for that severity and neurodata type."""
+    """
+    Wrap a check function with this decorator to add it to the check registry and automatically parse some output.
+
+    Parameters
+    ----------
+    importance : Importance
+        Importance has three levels:
+            CRITICAL
+                - potentially incorrect data
+            BEST_PRACTICE_VIOLATION
+                - very suboptimal data representation
+            BEST_PRACTICE_SUGGESTION
+                - improvable data representation
```

Comment:
Let's make sure this looks good in readthedocs

Reply:
https://nwbinspector--139.org.readthedocs.build/en/139/api/register_check.html

Reply:
That is, 
![image](https://user-images.githubusercontent.com/51133164/160415175-f559df57-852d-469f-9b7f-18b05272c57f.png)

which is the same (copied/pasted) as the appearance in the docstrings for the core functions.